dali/operators/image/color/color_twist.cc

```diff
@@ -100,6 +100,18 @@ Values >= 0 are accepted. For example:
     .AddParent("ColorTransformBase")
     .InputLayout(0, "HWC");
 
+
+template <>
+bool ColorTwistBase<CPUBackend>::CanInferOutputs() const  {
+  return false;
```

Comment:
Why it can't?

Reply:
I didn't implement `SetupImpl` .  It's small so I implemented it. Btw, I've also added  a test comparing GPU vs CPU color twist to be sure that after the changes all implementations are still compatible.